css/style.css

```diff
@@ -0,0 +1,248 @@
+body{
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+    display: block;
+
```

Comment:
I would suggest that you always remove empty lines from your CSS codebase also as it makes it readable and clean
Thanks 💯👏🙌